calculator.js

```diff
@@ -1,2 +1,31 @@
 var calculator = {
-}
+
+  add: function( num1, num2 ) {
+    console.log( num1 + num2 );
+  },
+
+  subtract: function(num1, num2) {
+    console.log( num1 - num2 );
+  },
+
+  multiply: function( num1, num2 ) {
+    console.log( num1 * num2 );
+   },
+
+  division: function( num1, num2) {
+    console.log(num1/num2);
+   },
+
+  exponent: function( num1, num2) {
+    console.log(Math.pow(num1, num2));
+  },
+
+  Modulus: function( num1, num2) {
+    console.log(num1 % num2);
+  }
+
+};
+
+var memory = function memVal() {
+    console.log(memVal());
+  };
```

Comment:
good work overall! 👍 👍 👍 

comment on lines 29-31:
when you are defining a function via function expression, the function is named by the variable. you're setting the value of a variable to an anonymous function. Example below:

```js
var someFunc = function(){ 
  //function code here 
}
```